src/plugin/models/signMessageModels.ts

```diff
@@ -0,0 +1,55 @@
+/** EthSignMethods enum contains the methods that can be used to sign a string */
+
+export enum SignMethod {
```

Comment:
This should be renamed to be clear is for signMessage only
SignMethod => SignMessageMethod